### DISCUSSION TITLE
Multiple keys pressed at the same time / key mashing support?

### DISCUSSION THREAD

**Discussion by silverbucket** [charmbracelet/bubbletea#1264] [Q&A]
Hi all, I'm using Bubbletea and Lipgloss to render a table. The user can use the arrow keys to navigate around the table cells. 
I would like to be able to either navigate diagonal when two direction buttons are pressed at the same time, or alternatively just detect that two buttons are being pressed (and held) and then move diagonally. Either a one-time thing, or a press and hold thing. 

It seems whichever key was the last to get pressed is the only event that comes through in the Update method. I expect that the user will be wanting to press the keys quickly, hold to move fast across this 2d plane generated table.

**Comment by caarlos0** [comment]
@aymanbagabas can correct me but I think this is coming in v2 with kitty keyboard
